docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4401,6 +4401,42 @@ Content-Type: text/plain; charset=utf-8
 -   **404** - plugin not installed
 -   **500** - plugin is active
 
+### Create a plugin
+
+`POST /plugins/(plugin name)/create`
+
+Create a plugin
+
+**Example request**:
+
+To create a plugin named `plugin`
+
+```
+POST /plugins/plugin:latest/create HTTP/1.1
+
+Header should be set to "Content-Type: application/tar"
```

Comment:
Since this is an example, the header should be part of the example;

```
POST /plugins/plugin:latest/create HTTP/1.1
Content-Type: application/tar

{% raw %}
{{ TAR STREAM }}
{% endraw %}
```

Also can you check if `application/tar` or `application/x-tar` is the correct header? We use `x-tar` in other endpoints; https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api_v1.25.md#extract-an-archive-of-files-or-folders-to-a-directory-in-a-container

According to http://superuser.com/a/960710, there's still no official content-type for tar